sql/2020/09_Performance/lh6_performance_score_distribution.sql

```diff
@@ -0,0 +1,14 @@
+#standardSQL
+# Distribution of LH6 performance score.
+
+SELECT
```

Comment:
Please apply formatting changes suggested in the LH5 query

Reply:
Cheers, done!